File: library-2/src/main/java/ru/otus/library/dto/Book.java
package ru.otus.library.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Book {

    private Long id;

    private String title;

    private Integer pages;

    private Genre genre;

    private Author author;

    private List<Comment> comments;
}

File: library-5.1/src/app/model/book.ts
import { Genre } from "./genre";
import { Author } from "./author";
import { Comment } from "./comment";

export class Book {
  id: string = "";
  title: string = "";
  pages: string = "";
  genre: Genre = new Genre();
  author: Author = new Author();
  comments: Array<Comment> = [];
}
